src/render/context/mod.rs

```diff
@@ -48,9 +57,14 @@ pub struct Context {
     /// Directory to traverse; defaults to current working directory
     dir: Option<PathBuf>,
 
-    /// Turn on colorization always
-    #[arg(short = 'C', long)]
-    pub force_color: bool,
+    /// Coloring of the Output:
+    /// Colored: (Default)
```

Comment:
Final change: Would you mind moving the doc comments for each enum variant over to the enum itself? `clap` knows how to handle the `--help` output when done in that fashion. Checkout `src/render/context/sort.rs` as a reference

Reply:
For someone new to Rust you have fantastic intuition. This is fine! I'll approve and merge this once tests pass. If I need to make any small adjustments to the documentation I can do that later. Thanks a bunch for the contribution!

Reply:
If I don't merge this within the next few minutes I'll merge it immediately after I get off work.